Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -768,34 +768,66 @@ extension Tensor where Scalar: TensorFlowFloatingPoint {
 // Padding
 //===------------------------------------------------------------------------------------------===//
 
+
 public extension Tensor where Scalar: Numeric {
-    /// Returns a padded tensor according to the specified padding sizes.
+    /// Padding modes:
+    ///  * `constant` - pad with constant value.
```

Comment:
* The doc comment for a type should be a descriptive noun phrase that explains what the type (or values of this type) represents.
* Wording-wise, it'd be nice to follow precedents. Here's one: https://developer.apple.com/documentation/realitykit/collisioncomponent/mode.
* Doc comments for cases should be defined on cases.

Suggestion:
```swift
    /// A mode that dictates how a tensor is padded.
    enum PaddingMode {
        /// Pads with a constant value.
        case constant(Scalar)
        /// Mirrors values along padding dimensions, excluding the edge value.
        case reflect
        /// Mirror values along padding dimensions, including the edge value.
        case symmetric
     }
```